Can't run a local dev build

Getting multiple can't resolves like this

```
ERROR in ./example/src/index.tsx 2:0-35 Module not found: Error: Can't resolve 'react-dom' in '/.../kbar-main/example/src'

ERROR in ./example/src/index.tsx 4:0-59 Module not found: Error: Can't resolve 'react-router-dom' in '/.../kbar- main/example/src'

ERROR in ./src/InternalKeyboardEvents.tsx 1:0-31 Module not found: Error: Can't resolve 'react' in '/.../kbar-main/src' 
@ ./src/KBarContextProvider.tsx 3:0-62 9:41-63 
@ ./example/src/App.tsx 23:0-61 42:45-57 
@ ./example/src/index.tsx 3:0-24 5:153-156
```

Do you know what might be happening?

--- a/package.json
+++ b/package.json
@@ -15,7 +15,13 @@
     "typescript": "^4.4.2",
     "webpack": "^5.51.1",
     "webpack-cli": "^4.8.0",
-    "webpack-dev-server": "^4.1.0"
+    "webpack-dev-server": "^4.1.0",
+    "@reach/portal": "^0.16.0",
+    "fast-equals": "^2.0.3",
+    "match-sorter": "^6.3.0",
+    "react-router-dom": "^5.3.0",
+    "react": "^16.0.0 || ^17.0.0",
+    "react-dom": "^16.0.0 || ^17.0.0"
   },
   "main": "lib/index.js",
   "scripts": {
@@ -23,7 +29,10 @@
     "build:example": "webpack --config ./example/webpack.prod.cjs",
     "dev": "webpack-dev-server --config ./example/webpack.config.cjs --hot"
   },
-  "keywords": [],
+  "keywords": [
+    "command bar",
+    "search"
+  ],
   "author": "Tim Chang <timchang.tcc@gmail.com>",
   "license": "MIT",
   "repository": {
@@ -31,11 +40,7 @@
     "url": "https://github.com/timc1/kbar.git"
   },
   "peerDependencies": {
-    "@reach/portal": "^0.16.0",
-    "fast-equals": "^2.0.3",
-    "match-sorter": "^6.3.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.3.0"
+    "react": "^16.0.0 || ^17.0.0",
+    "react-dom": "^16.0.0 || ^17.0.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,12 @@
 {
   "name": "kbar",
   "version": "0.1.0-beta.1",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -rf ./lib & tsc",
+    "build:example": "webpack --config ./example/webpack.prod.cjs",
+    "dev": "webpack-dev-server --config ./example/webpack.config.cjs --hot"
+  },
   "devDependencies": {
     "@types/react": "^17.0.19",
     "@types/react-dom": "^17.0.9",
@@ -16,18 +22,18 @@
     "webpack": "^5.51.1",
     "webpack-cli": "^4.8.0",
     "webpack-dev-server": "^4.1.0",
-    "@reach/portal": "^0.16.0",
-    "fast-equals": "^2.0.3",
-    "match-sorter": "^6.3.0",
     "react-router-dom": "^5.3.0",
     "react": "^16.0.0 || ^17.0.0",
     "react-dom": "^16.0.0 || ^17.0.0"
   },
-  "main": "lib/index.js",
-  "scripts": {
-    "build": "rm -rf ./lib & tsc",
-    "build:example": "webpack --config ./example/webpack.prod.cjs",
-    "dev": "webpack-dev-server --config ./example/webpack.config.cjs --hot"
+  "dependencies": {
+    "@reach/portal": "^0.16.0",
+    "fast-equals": "^2.0.3",
+    "match-sorter": "^6.3.0"
+  },
+  "peerDependencies": {
+    "react": "^16.0.0 || ^17.0.0",
+    "react-dom": "^16.0.0 || ^17.0.0"
   },
   "keywords": [
     "command bar",
@@ -38,9 +44,5 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/timc1/kbar.git"
-  },
-  "peerDependencies": {
-    "react": "^16.0.0 || ^17.0.0",
-    "react-dom": "^16.0.0 || ^17.0.0"
   }
 }

